recipes/mpitrampoline/meta.yaml

```diff
@@ -0,0 +1,74 @@
+{% set name = "MPItrampoline" %}
+{% set version = "5.5.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://github.com/eschnett/{{ name }}/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: bdfb145b754cf13d38485808f79a5638d954dbe181e7bb53a1950c71deed6989
+
+build:
+  # Fortran compiler is not found on Windows
+  skip: true  # [win]
+  number: 0
+
+requirements:
+  build:
+    - cmake
+    - ninja
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - {{ compiler('fortran') }}
+    - {{ stdlib('c') }}
+  host:
+    - pip
+    - python
+    - setuptools
```

Comment:
The documentation for your packge states that it wraps around an actual MPI implementation. Doesn't that mean it needs to dynamically link to said implementation? So openmpi should be listed in the host deps or is it a dlopen situation?

https://conda-forge.org/docs/maintainer/knowledge_base/#message-passing-interface-mpi